src/lib/queries/fetchTopRewards.ts

```diff
@@ -25,16 +14,28 @@ export async function fetchTopRewards(): Promise<LeaderboardEntry[]> {
 	});
 	const data: TopAccountsQuery = (await response.json()).data;
 
-	return (data.accountsByCyBalance ?? []).map((account) => {
-		const sharePercentage = account.totalCyBalance / data.eligibleTotals?.totalEligibleSum;
+	const eligibleTotals = data.eligibleTotals;
+	if (!eligibleTotals) throw 'No eligible totals';
```

Comment:
single line `if` statements are undebuggable without restructuring because they introduce meaningful whitespace, so it's fragile in development

if the linter isn't forcing this i'd use `{}`

Reply:
done